tests/rptest/tests/controller_upgrade_test.py

```diff
@@ -63,17 +57,26 @@ def cluster_is_stable():
 
             return True
 
-        # reset environment to use latest redpanda logical version
-        self.redpanda.set_environment({})
-        for n in self.redpanda.nodes:
+        # Get to a stable starting point before asserting anything about our
+        # workload.
+        wait_until(cluster_is_stable, 90, backoff_sec=2)
+
+        admin_fuzz.wait(10, 240)
+        num_executed_before_restart = admin_fuzz.executed
 
-            self.redpanda.restart_nodes(n, stop_timeout=60)
+        # reset to use latest Redpanda version
+        installer.install(self.redpanda.nodes, RedpandaInstaller.HEAD)
+
+        for n in self.redpanda.nodes:
+            # NOTE: we're not doing a rolling restart to avoid interfering with
+            # the admin operations.
```

Comment:
nit: Is there no value in upgrade testing while admin operations are running concurrently? Could this be a test case later in the future?

Reply:
Yeah that's a good idea. Done